web/app/themes/radcliffe/report.php

```diff
@@ -1,9 +1,6 @@
 <?php
-
 get_header();
-
 global $td;
-
 ?>
 
 <div class="content">
```

Comment:
The element 'main' is missing from this file. Was this intentional? If so, would you kindly explain?

Reply:
Huh? Compared with master and no changes.